feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -0,0 +1,158 @@
+                isValid = false,
+                supportTextError = stringResource(id = R.string.supportError_Unit),
+                onValueChange = {}
+            )/* TODO - Change the component when available (Disable/Generic) */
+            LPPhoneNumberText(
+                modifier = Modifier.padding(
+                    top = eSdp10
+                ),
+                value = numberValue,
+                onPhoneChange = {
+                    numberValue = it
+                }
+            )
+            /*LpOutlinedTextFieldNumber(
+                modifier = Modifier.padding(
+                    top = eSdp10
+                ),
+                label = stringResource(id = R.string.label_ContactPhone),
+                hint = stringResource(id = R.string.hint_ContactPhone),
+                onValueChange = {}
+            )*/
+            DropdownTextField(
+                modifier = Modifier.padding(
+                    top = eSdp10
+                ),
+                title = stringResource(id = R.string.label_PetInUnit),
+                items = arrayPetInUnit.toList(),
+                selected = {}
+            )
+            Text(
+                text = stringResource(id = R.string.text_Service),
+                modifier = Modifier.padding(
+                    top = eSdp10
+                ),
+                style = LPTypography.bodyLarge
+            )
+            DropdownTextField(
+                modifier = Modifier.padding(
+                    top = eSdp06
+                ),
+                title = stringResource(id = R.string.label_Category),
+                items = arrayCategory.toList(),
+                selected = {}
+            )
+            LpOutlinedTextFieldMail(
+                modifier = Modifier
+                    .padding(
+                        top = eSdp10
+                    )
+                    .height(height = eSdp104)
+                    .fillMaxWidth(),
+                label = stringResource(id = R.string.label_Description),
+                hint = stringResource(id = R.string.hint_Description),
+                isValid = isValidDescription,
+                supportTextError = stringResource(id = R.string.supportError_Description),
+                onValueChange = {
+                    isValidDescription = it.isEmpty() || it.matches(Regex(".*[a-zA-Z]+.*")).not()
+                }
+            )
+            Text(
+                text = stringResource(id = R.string.text_Video),
+                modifier = Modifier.padding(
+                    top = eSdp10
+                ),
+                style = LPTypography.bodyLarge
+            )
+            LpFileButton(
+                modifier = Modifier
+                    .padding(
+                        top = eSdp06
+                    )
+                    .fillMaxWidth(),
+                mimeTypes = arrayOf("Option A", "Option B"),
+                onFileSelected = {}
```

Comment:
Can you add a "TODO" to know how it's going to work

Reply:
Corrections were made. Thank you @carlosph816.